### ISSUE TITLE
Question about zero-shot

### ISSUE THREAD

**Issue by jasperhyp** [YyzHarry/multi-domain-imbalance#2]
Hi!  Thanks for the great work!  I was wondering how is the zero-shot domain-class pair's mean representation $\mu_{d, c}$ calculated.  Was not able to find details in the paper.  Thanks!

**Comment by YyzHarry** [comment]
Hi - thanks for your interest! For zero-shot domain-class pairs, during training we do not calculate their $\mu_{d,c}$ (as there's no sample). Correspondingly, BoDA loss only involves domain-class pairs that have samples in the training set.

**Comment by jasperhyp** [comment]
Oh I see, that's really interesting.  Thanks for the clarification!
